### PR TITLE
MGMT-3458 Pass REST API params to FIO cmd

### PR DESCRIPTION
**Pull request by rollandf**

### CODE REVIEW

**Comment by rollandf** [comment]
/hold
Verifying e2e

**Comment by rollandf** [comment]
```
Jan 04 14:25:40 test-infra-cluster-assisted-installer-master-0 next_step_runne[1880]: time="04-01-2021 14:25:40" level=info msg="Executing step: <install-0160b340>, command: <bash>, args: <[-c dd if=/dev/zero of=/dev/sr0 bs=512 count=1 ; podman run --privileged --net=host --rm --quiet -v /dev:/dev:rw -v /var/log:/var/log -v /run/systemd/journal/socket:/run/systemd/journal/socket --env PULL_SECRET_TOKEN --env HTTP_PROXY --env HTTPS_PROXY --env NO_PROXY --env http_proxy --env https_proxy --env no_proxy quay.io/rollandf/bm-inventory:fioagent fio_perf_check --url http://10.1.37.42:6000 --cluster-id cf748dfd-f22e-4cdb-9afa-e374cc373d9d --host-id 1c5c4c6f-8eb0-4397-9659-d0acb8fb3169 --agent-version quay.io/rollandf/bm-inventory:fioagent --insecure true \"{\\\"duration_threshold\\\":1000,\\\"exit_code\\\":222,\\\"path\\\":\\\"/dev/vda\\\"}\" ; podman run --privileged --pid=host --net=host --name=assisted-installer --volume /dev:/dev:rw --volume /opt:/opt:rw --volume /var/log:/var/log:rw --volume /run/systemd/journal/socket:/run/systemd/journal/socket --env=PULL_SECRET_TOKEN quay.io/ocpmetal/assisted-installer:latest --role master --cluster-id cf748dfd-f22e-4cdb-9afa-e374cc373d9d --host-id 1c5c4c6f-8eb0-4397-9659-d0acb8fb3169 --boot-device /dev/vda --url http://10.1.37.42:6000 --openshift-version 4.6 --high-availability-mode Full --mco-image quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:01a5b96ce75867875d08a1a81807f3503068248d6093ef6669d0b50675f4816f --controller-image quay.io/ocpmetal/assisted-installer-controller:latest --agent-image quay.io/ocpmetal/assisted-installer-agent:latest --insecure true]>" file="step_processor.go:78" request_id=62de1e98-86e0-41f7-9265-6d59694dbf4b

Jan 04 14:26:00 test-infra-cluster-assisted-installer-master-0 fio_perf_check[4069]: time="04-01-2021 14:26:00" level=info msg="FIO result on disk /dev/vda :fdatasync duration 5 ms , threshold: 1000 ms" file="fio_perf_check.go:50"
```

File: internal/host/fioperfcheckcmd.go [hunk 1]
```diff
@@ -16,19 +17,26 @@ const (
...
 
+type FioPerfCheckConfig struct {
+	ServiceBaseURL       string
+	ClusterID            string
+	HostID               string
+	UseCustomCACert      bool
+	FioPerfCheckImage    string
+	SkipCertVerification bool
+	Path                 string
+	DurationThreshold    int64
```
Comment (avishayt) [inline internal/host/fioperfcheckcmd.go:28]: Please add units to the variable name (also on the API where the agent returns the value to the service)

Comment (rollandf) [reply internal/host/fioperfcheckcmd.go]: Done

**Pull request merged**